SoftwareCo/SoftwareCo/SoftwareCoPackage.cs

```diff
@@ -697,11 +699,11 @@ private static async Task<SessionSummaryResult> GetSessionSummaryStatusAsync(boo
                     updateStatusBarWithSummaryData();
                 }
 
-            }

```

Comment:
What is the reason to remove this logic here?

Reply:
Reason to remove the check of SessionsummaryFile exist or not is
As we right sessionsummary.json file when we increment it by **one minute** in storepayload method ( inside check we also having **_sessionSummary.currentDayMinutes == 0** check)
or when we sendoffline data every **30 minutes** .
So at first call, we will not go inside that check and we won't be fetching values from "/sessions/summary" API,
We need to wait for 30 minutes to get the next call.  
Now on the first call, we fetch sessionSummaryData from API and write it into the file.
If I am missing anything please let me know.